tests/hpc/slurm_master_db.pm

```diff
@@ -18,6 +18,165 @@ use warnings;
 use testapi;
 use lockapi;
 use utils;
+use Data::Dumper;
+use Mojo::JSON;
+
+## TODO: provide better parser for HPC specific tests
+sub get_test_name {
+    return my $name = (caller(2))[3];
+}
+
+sub validate_result {
+    my ($result) = @_;
+
+    if ($result == 0) {
+        return 'PASS';
+    } elsif ($result == 1) {
```

Comment:
what about plenty of cases when command returning some sane error code ? you want all of them end up in undef ?

Reply:
Sure, this is super simplistic first iteration thing. I will of course keep improving this.

Overall I aim at having external repo with some .sh tests and then I might look into reusing ltp test runner for this etc.